combat-trainer.lic

```diff
@@ -1641,6 +1642,14 @@ class GameState
 
     @use_weak_attacks = settings.use_weak_attacks
     echo("  @use_weak_attacks: #{@use_weak_attacks}") if $debug_mode_ct
+
+    @use_analyze_flame = settings.use_analyze_flame
```

Comment:
1. This could be generalized to support any Expertise-specific analyze by making the new setting a string instead of a bool. nil means we do not analyze, anything else is used as part of the analyze command. I'd be in favor of making this change.

2. We could even make this usable by non-Barbarians by having a bool *and* a string. The bool is whether or not to use analyze, the string is the type, and non-Barbarians use nil for the string and true for the bool. I'm inclined to skip this unless someone requests it.